build-yaml/econet-etwh-esp32.yaml

```diff
@@ -0,0 +1,13 @@
+---
+substitutions:
```

Comment:
I was going to suggest to have 2 files: esp32 and esp8266 and include the appropriate one in each file but what you have might be better. If you adopt with what you have, will you get the substitutions as is here? If you were including esp32.yaml I assume you would just see the include line? I think it would be better to have the substitutions so that people can adjust them if needed.

Reply:
I thought about that but wasn't sure how conflicting substitutions would be handled from different sub-files

But yes, in the end I figured there wasn't _that much_ duplicated material and it was useful to see those key settings in each file anyway.

The real fix would be for ESPHome to allow some dynamism in configuration (e.g. with conditionals) but it sounds like that work has been punted.